includes/wikia/tests/UploadVerifyFileTest.php

```diff
@@ -48,6 +48,7 @@ public function uploadVerifyFileDataProvider() {
 				'uploadContent' => base64_decode('R0lGODlhAQABAIABAAAAAP///yH5BAEAAAEALAAAAAABAAEAQAICTAEAOw%3D%3D'), // blank 1x1 gif
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)